cmd/fyne_demo/tutorials/dialog.go

```diff
@@ -143,6 +144,32 @@ func dialogScreen(win fyne.Window) fyne.CanvasObject {
 				log.Println("Please Authenticate", username.Text, password.Text)
```

Comment:
I just tested this locally. I think you could remove the "Custom Dialog (Login Form)". Your new dialog is basically doing the same thing, but better :)

Reply:
Okay, will do.